### ISSUE TITLE
Didn't charge as expected

### ISSUE THREAD

**Issue by rindlerblabla** [jonasbkarlsson/ev_smart_charging#124]
## Version of the custom_component

Version 0.9.48

## Describe the bug
I have the "EV Smart Charging Continuous charging preferred"-switch turned off and last night it was supposed to charge between 23-00 and then from 01-06, if I remember the graph. However it just charged the first hour ending up with an almost empty battery. Before this I have never had any issues similar to this and have used the service since the beginning (thanks for creating it!).

I have the smart charging activated and the opportunistic switch is turned off. It is somehow hard to debug why it didn't continue charging but maybe you can help out in what info you need.

For some reason the sensor both turn on/off at the same time at 00:00, and both the automation to turn off (lower the Easee dynamic limit to 0A) and turn on (set the dynamic limit to 16A) run at the same time, which probably caused the charging to stop.

```
EV Smart Charging Charging slogs av
06:00:00 - För 7 timmar sedan
EV Smart Charging Charging slogs på
00:00:00 - För 13 timmar sedan
EV Smart Charging Charging slogs av
00:00:00 - För 13 timmar sedan
2 februari 2023
EV Smart Charging Charging slogs på
23:00:00 - För 14 timmar sedan
```

**Comment by jonasbkarlsson** [comment]
Thanks for reporting this. Most likely, at 00, the integration understood that it would not be enough to schedule 01-06, and changed it to 00-06. And to turn off and then quickly turn on again didn't work as expected in your case.

This is weakness of the implementation that I'm aware of, but I haven't had the time and motivation to improve it. But now I have motivation. 😄 

The workaround to avoid this problem, until the issue is solved,  would be to set `switch.ev_smart_charging_continuous_charging_preferred` = `on`.

**Comment by rindlerblabla** [comment]
Thanks for your quick reply! Will turn the switch on for the time being.

**Comment by jonasbkarlsson** [comment]
This issue should now be fixed by #136 and is included in [v1.7.0](https://github.com/jonasbkarlsson/ev_smart_charging/releases/tag/v1.7.0).
